hedera-mirror-rest/model/customFeeLimits.js

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import {proto} from '@hashgraph/proto';
+import FixedFee from './fixedFee.js';
+
+class CustomFeeLimits {
+  /**
+   * Parses an array of serialized CustomFeeLimit messages from transaction.max_custom_fees
+   * @param {Buffer[]} customFeeLimits - An array of byte arrays representing serialized CustomFeeLimit messages.
+   */
+  constructor(customFeeLimits) {
+    if (!Array.isArray(customFeeLimits) || customFeeLimits.length === 0) {
+      this.fees = [];
+      return;
+    }
+    this.fees = (customFeeLimits ?? []).map((feeBytes) => {
+      const customFeeLimitSet = proto.CustomFeeLimit.decode(feeBytes);
+      return {
+        accountId: customFeeLimitSet.accountId,
+        fixedFees: (customFeeLimitSet.fees ?? []).map((fee) => new FixedFee(fee)),
+      };
```

Comment:
the previous comment about `FixedFee` model is not addressed. In short, we can't use `FixedFee`, and we don't have to convert it at all, just have to deserialize the protobuf bytes

```suggestion
   proto.CustomFeeLimit.decode(feeBytes)
```